tests/integration/questionnaire/test_questionnaire_page_titles.py

```diff
@@ -100,3 +100,12 @@ def test_content_page_should_use_nested_content_text_in_page_title_if_it_exists(
         self.assertEqualPageTitle(
             "This is the content title … - Interstitial Page Titles"
         )
+
+    def test_plural_page_title_should_use_other_title(self):
+        self.launchSurvey("test_plural_forms")
+
+        self.post({"number-of-people-answer": 0})
+
+        self.assertEqualPageTitle(
+            "… people live here, is this correct? - Test Plural Forms"
+        )
```

Comment:
We should also have integration tests for the summary content (question and answer).